rlkit/core/rl_algorithm.py

```diff
@@ -565,7 +450,10 @@ def evaluate(self, epoch):
             # TODO: put these lines into method evaluate_task(idx) that goes into sac.py?
             for _ in range(10):
                 self.infer_posterior(idx)
-                paths += self.eval_sampler.obtain_samples(num_samples=self.max_path_length + 1, deterministic=True, resample=np.inf)
+                # why do we want max_path_length + 1?
```

Comment:
this was a holdover from the way rlkit originally wrote `obtain_samples`, and the +1 was to fix an offset so you actually got a full traj

Reply:
I think we can kill the +1 now?